DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py

```diff
@@ -50,17 +51,24 @@
 process.DQMStore.referenceFileName = referenceFileName
 process = customise(process)
 process.DQMStore.verbose = 0
-process.source.minEventsPerLumi=100
```

Comment:
you might want to restore this one

Reply:
Hi Fede, thanks for catching this, should be fixed now.